tiles/src/main/java/com/protomaps/basemap/layers/Places.java

```diff
@@ -210,15 +212,30 @@ static int getSortKey(double minZoom, int kindRank, int populationRank, long pop
 
   private static final ZoomFunction<Number> LOCALITY_GRID_SIZE_ZOOM_FUNCTION =
     ZoomFunction.fromMaxZoomThresholds(Map.of(
-      6, 32,
-      7, 64
+      3, 24,
+      4, 24,
+      5, 24,
+      7, 24,
+      8, 32,
+      9, 32,
```

Comment:
I have noticed that some features are present in zoom 7, then they disappear at zoom 8, and reappear at zoom 9:

Example Toulon

Zoom 7 many points in orange circles:

![image](https://github.com/user-attachments/assets/db581b95-d600-4b68-9766-0d27e9acb9bc)

Zoom 8 only one point for Toulon:

![image](https://github.com/user-attachments/assets/492a4b3c-3097-48c6-b058-053b8cbd674e)


Zoom 9 many points reappear:

![image](https://github.com/user-attachments/assets/bfc9106f-d6f2-4188-836b-41febe260c95)

Reply:
Maybe this is due to the increase in the label grid cell size. Larger cell means more features fall into the same cell means fewer are retained...